src/commands/build.ts

```diff
@@ -44,11 +59,6 @@ export class BuildCommand implements Command {
           'additional `<link rel="prefetch">` tags into ' +
           'entrypoint and `<link rel="import">` tags into fragments and shell.'
     },
-    {
-      name: 'html.collapseWhitespace',
```

Comment:
Actually, collapseWhitespace is only one of many options offered here: https://www.npmjs.com/package/html-minifier

Are we now mapping minify to a specific set of those?